src/bittybuzz/include/bittybuzz/BittyBuzzVm.h

```diff
@@ -43,9 +43,11 @@ class BittyBuzzVm : public IBittyBuzzVm {
 
     bool step() override;
 
-    bbzvm_state getSate() const override;
+    const char* getState() const override;
```

Comment:
Shouldn't getState return the bbzm_state? If not, what's the difference between getState and getStateString?

Reply:
Reverted to the enum instead